src/main/java/heronarts/lx/color/LXPalette.java

```diff
@@ -505,15 +505,26 @@ private void _transitionColor(LXDynamicColor color, int i, float lerp) {
     case HSV:
       float fromHue = fromColor.hue.getValuef();
       float toHue = toColor.hue.getValuef();
+      float fromSat = fromColor.saturation.getValuef();
+      float toSat = toColor.saturation.getValuef();
+      float fromBright = fromColor.brightness.getValuef();
```

Comment:
No need for `fromBright`/`toBright` here...

Reply:
Done.  Equalized the saturations here as well.